x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/rare_view/detector_description.tsx

```diff
@@ -63,49 +58,82 @@ function createDetectorDescription(jobCreator: RareJobCreator, detectorType: RAR
   const populationFieldName = jobCreator.populationField?.id;
   const splitFieldName = jobCreator.splitField?.id;
 
-  const beginningSummary = i18n.translate(
-    'xpack.ml.newJob.wizard.pickFieldsStep.rareField.plainText.beginningSummary',
+  const rareSummary = i18n.translate(
```

Comment:
The summaries shown in the final summary step of the wizard, before creating the job, also need amending to follow the new patterns. For example, this is a population job, no partition field:

Detector step:
![image](https://user-images.githubusercontent.com/7405507/125828290-6b5b0429-e496-4643-bb57-5a9e350fa8ce.png)

But in the summary step this is shown:
![image](https://user-images.githubusercontent.com/7405507/125828343-c1df527e-85fc-42b2-98ff-6f761012b641.png)

Reply:
It seems to me that this problem occurs in https://github.com/elastic/kibana/blob/master/x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/rare_view/rare_view.tsx. It doesn't seem be be sending the current detector type to `RareDetectorsSummary`. For example, if I put RARE_DETECTOR_TYPE.RARE_POPULATION instead of RARE_DETECTOR_TYPE.RARE in [rare_view.tsx#L29](https://github.com/elastic/kibana/blob/86225685c6ee559c8b669658dd2f6b05024dfba5/x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/rare_view/rare_view.tsx#L29), for example, it's using that value instead.

Reply:
This will be fixed by https://github.com/elastic/kibana/pull/105966